README.md

```diff
@@ -86,9 +87,9 @@ You can install the toolbox through PyPI with:
 ```
 pip install POT
 ```
-or get the very latest version by downloading it and then running:
+or get the very latest version by running:
 ```
-python setup.py install --user # for user install (no root)
+pip install git+https://github.com/PythonOT/POT.git # with --user for user install (no root)
```

Comment:
```suggestion
pip install -U https://github.com/PythonOT/POT/archive/master.zip # with --user for user install (no root)
```

it will also work on windows without git installed.